### DISCUSSION TITLE
acess a service like database on running host

### DISCUSSION THREAD

**Discussion by boks21** [rancher-sandbox/rancher-desktop#1745] [General]
hello all,

there is a way to acess a postgress database running in the host machine (windows) with rancher desktop? Or I can forward all networks requests to the host? ie, on windows I can connect to a remote database on 143.x.x.x, and would like to connect from container too.

**Comment by jandubois** [comment]
Using Rancher Desktop 1.1.1 you should be able to use `host.docker.internal` to connect to the host from inside a container.
